web_tree_dynamic_colored_field/static/src/js/web_tree_dynamic_colored_field.js

```diff
@@ -61,6 +81,8 @@ odoo.define('web_tree_dynamic_colored_field', function (require) {
             }
             this.applyColorizeHelper($td, nodeOptions, node, 'fg_color', 'color', ctx);
             this.applyColorizeHelper($td, nodeOptions, node, 'bg_color', 'background-color', ctx);
+            this.applyDirectColorizeHelper($td, nodeOptions, node, 'fg_color_field', 'color', ctx);
```

Comment:
this would be `color_field`, right?

Reply:
No, this is a color_field selected for the column and line (background_color is for all the items of the line).
Sometimes it is hard to create the specifications on a single line and you would like to compute it with python. It is used inside options of the field.